### ISSUE TITLE
Question : any specific reason you set .requires_grad_ field of BatchNorm1d bias before embedding in every model to "False"?

### ISSUE THREAD

**Issue by dazzle-me** [SeuTao/Kaggle_Whale2019_2nd_palce_solution#9]
Why won't let model learn bias by itself via gradient descent, or it is somehow related to ensembling?
I haven't encountered such design yet and want to know logic behind this concept, can you explain?
